cdp_scrapers/tests/test_scrapers.py

```diff
@@ -24,10 +24,10 @@
         (
             datetime(2022, 11, 7),
             datetime(2022, 11, 16),
-            2,
-            33,
+            3,
+            31,
```

Comment:
These are valid changes to test answers. The old expected values of 2 events and 31 items in the first event seem to be for the 11/15/22 event. The new answer is for the 11/08/22 event, which is earlier in the date range than 11/15/22.